package.json

```diff
@@ -89,15 +89,15 @@
     "puppeteer": "1.13.0",
     "rcfile": "1.0.3",
     "read-pkg-up": "4.0.0",
-    "rollup": "1.6.0",
+    "rollup": "1.7.0",
     "rollup-plugin-babel": "4.3.2",
     "rollup-plugin-commonjs": "9.2.1",
-    "rollup-plugin-json": "3.1.0",
+    "rollup-plugin-json": "4.0.0",
```

Comment:
https://github.com/rollup/rollup-plugin-json/releases/tag/v4.0.0
Is there an easy way to see if this doesn't break anything?

Reply:
Thanks for asking.

> Pass all JSON data through dataToEsm to consistently support "compact" formatting, support empty keys, abandon Node 4 support, add prettier, update dependencies (#53)

I presumed it's about dropping node@4 support which shouldn't be a big problem.